haskell/toolchain.bzl

```diff
@@ -96,6 +96,7 @@ def _run_ghc(hs, cc, inputs, outputs, mnemonic, arguments, params_file = None, e
 
     hs.actions.run_shell(
         inputs = inputs,
+        input_manifests = input_manifests,
```

Comment:
And this?

Reply:
It's an argument to `run_shell()`, so it's documented in the [API documentation](https://docs.bazel.build/versions/master/skylark/lib/actions.html#run_shell) for `run_shell()`.

Reply:
Nevermind. I was just surprised to discover new arguments to `run_shell` and `_run_ghc` which were not used elsewhere in this PR.

Reply:
Ah, you're right. They're used in a followup PR (not submitted yet). Though that extra argument does no harm at this point.